internal/downloader/downloader.go

```diff
@@ -37,7 +43,41 @@ type downloadImpl interface {
 	Download(context.Context, string, []string) error
 }
 
-var gatherFunc = gather.Gather
+var log = logrus.StandardLogger()
+
+var gatherFunc = func(ctx context.Context, source, destination string) (metadata.Metadata, error) {
+	initialize()
+	return gather.Gather(ctx, source, destination)
+}
+
+var _initialize = func() {
```

Comment:
Could this be done during `init` to avoid having to use `sync.OnceFunc`?

Reply:
Could not control it in tests then